setup.py

```diff
@@ -19,9 +21,21 @@
       url='https://github.com/AtteqCom/zsl',
       license='MIT',
       install_requires=[
-        'flask_injector==0.8.0',
-        'future'
+          'future',
+          'Flask-Injector>=0.8' if _is_py3 else 'Flask-Injector==0.8.0',
+          'requests>=2.7',
+          'SQLAlchemy>=1',
```

Comment:
Well. when we'll make it so, it will not be required for now, I would not be solving it in this issue.